src/v2/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -116,54 +131,89 @@ export class TransactionDetailsSummaryItem extends React.Component<
       : this.props.order.myLastOffer
   }
 
+  amountPlaceholder = () => {
+    const { transactionStep } = this.props
+
+    if (transactionStep === "review") {
+      return this.avalaraPhase2enabled
+        ? "Waiting for final costs"
+        : "To be confirmed*"
+    }
+
+    if (transactionStep === "payment") {
+      return this.avalaraPhase2enabled ? "Waiting for final costs" : "—"
+    }
+
+    return this.avalaraPhase2enabled ? "Calculated in the next steps" : "—"
+  }
```

Comment:
#nit-picking. My experience to make removing the feature flag easier in the future is to wrap the condition as outside as possible so we can simply delete a few lines. For example (not tested),
```typescript
amountPlaceholder = () => {
  const { transactionStep } = this.props

  if (this.avalaraPhase2enabled) {
    return ["review", "payment"].includes(transactionStep) ? "Waiting for final costs" : "—"
  }

  return transactionStep === "review" ? "To be confirmed*" : "—"
}
```

Reply:
This is so much cleaner thanks 🙏